website/docs/d/event_orchestration_integration.html.markdown

```diff
@@ -0,0 +1,41 @@
+---
+layout: 'pagerduty'
+page_title: 'PagerDuty: pagerduty_event_orchestration_integration'
+sidebar_current: 'docs-pagerduty-datasource-event-orchestration-integration'
+description: |-
+  Get information about an Integration for an Event Orchestration that you have created.
+---
+
+# pagerduty_event_orchestration_integration
+
+Use this data source to get information about a specific [Integration][1] for an Event Orchestration.
+
+## Example Usage
+
+```hcl
+
+data "pagerduty_event_orchestration_integration" "integration" {
+  event_orchestration = "19acac92-027a-4ea0-b06c-bbf516519601"
+  id = "1b49abe7-26db-4439-a715-c6d883acfb3e"
+  label = "Example integration"
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `event_orchestration` - (Required) ID of the Event Orchestration to which this Integration belongs to.
+- `id` - (Optional) ID of the Integration associated with the Event Orchestration. Specify either `id` or `label`. If both are specified `id` takes precedence.
+- `label` - (Optional) Name/description of the Integration associated with the Event Orchestration. Specify either `id` or `label`. If both are specified `id` takes precedence. The value of `label` is not unique and potentially there might be multiple Integrations with the same `label` value associated with the Event Orchestration.
```

Comment:
Can we also add that if an EO has more than one integrations with the given label, Terraform plan will return an error containing the ambiguous label.

Reply:
Added more explanation in https://github.com/PagerDuty/terraform-provider-pagerduty/pull/646/commits/cc87c93c2a79c8a5e3d72d9b1b58c3854b56aae0